pynxtools/dataconverter/readers/xps/reader.py

```diff
@@ -48,41 +56,25 @@
 CONVERT_DICT = {
     "unit": "@units",
     "version": "@version",
-    "User": "USER[user]",
-    "Instrument": "INSTRUMENT[instrument]",
-    "Source": "SOURCE[source]",
-    "Beam": "BEAM[beam]",
-    "Analyser": "ELECTRONANALYSER[electronanalyser]",
-    "Collectioncolumn": "COLLECTIONCOLUMN[collectioncolumn]",
-    "Energydispersion": "ENERGYDISPERSION[energydispersion]",
-    "Detector": "DETECTOR[detector]",
-    "Manipulator": "MANIPULATOR[manipulator]",
-    "Process": "PROCESS[process]",
-    "Sample": "SAMPLE[sample]",
-    "Data": "DATA[data]",
+    "user": "USER[user]",
+    "instrument": "INSTRUMENT[instrument]",
```

Comment:
This relates to the comment for the `eln_data.yaml` file. It should be correctly translated as outputed by nomad.